examples/navigation/epics/adminAccess.js

```diff
@@ -0,0 +1,15 @@
+import { push, LOCATION_CHANGE } from 'react-router-redux';
+
+function hasAccess() {
+  return false;
+}
+
+export default function adminAccess(action$) {
+  return action$.ofType(LOCATION_CHANGE)
+    .map(action => action.payload.pathname)
+    .filter(pathname => pathname === '/admin')
+    .map(hasAccess)
+    .filter(access => !access)
```

Comment:
I don't understand the purpose of these two lines:

```js
.map(hasAccess) // hasAccess always returns false.
.filter(access => !access) // ...so this will allow everything through
```

They seem like they can be omitted.